src/components/erc20/marketplace/order_book.tsx

```diff
@@ -132,7 +144,7 @@ class OrderBookTable extends React.Component<Props> {
                                     borderTop: true,
                                 }}
                             >
-                                {spread.toFixed(UI_DECIMALS_DISPLAYED_PRICE_ETH)}
+                                {percentageSpread.toFixed(0)}%
```

Comment:
Req: I think 0 decimal places won't make sense most of the time. I think 2 should work for the time being.

Reply:
Done